src/components/Card/Card.component.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import Card from './Card.component';
+
+const cardMock = {
+  title: 'Title',
+  description: 'Description',
+  url: 'https://i.ytimg.com/vi/HYyRZiwBWc8/mqdefault.jpg',
+};
+
+describe('component Card', () => {
+  it(' display Card elements', () => {
+    render(<Card {...cardMock} />);
+
+    expect(screen.getByText(cardMock.title)).toBeTruthy();
```

Comment:
For the card you could add `data-testid` and the compare the text with the one you provided like:

```javascript
<CardTitle data-testid='card-title'>{title}</CardTitle>
```

and in your test:

```javascript
const title = screen.getByTestId('card-title')
expect(title.textContent).toEqual(cardMock.title);
```

This is useful for targeting the specific component you want to test.